skillmap/src/components/makecodeFrame.tsx

```diff
@@ -195,6 +195,17 @@ class MakeCodeFrameImpl extends React.Component<MakeCodeFrameProps, MakeCodeFram
             }
         };
 
+        // Patch the name, otherwise it will be the name of the GitHub repo
+        if (project.header?.name !== title) {
+            project.header!.name = title;
+            const pxtJSON = project.text!["pxt.json"];
+            if (pxtJSON) {
+                const config = JSON.parse(pxtJSON);
+                config.name = title;
```

Comment:
do we need a null check for config if the JSON.parse fails, or is this pretty safe?

Reply:
I think it's safe? if pxt.json isn't there then I think we have other problems

Reply:
oh yeah good point